app/controllers/volunteer_ops_controller.rb

```diff
@@ -57,17 +57,35 @@ def volunteer_op_params
 
   private
 
-  def build_map_markers(organisations)
-    ::MapMarkerJson.build(organisations) do |org, marker|
+  def assemble_volunteer_markers(organisations)
+    harrow_markers = build_map_markers(organisations)
+    doit_markers = '[]'
+    if Feature.active? :doit_volunteer_opportunities
+      doit_orgs = collect_all_items
+      doit_markers = build_map_markers(doit_orgs, :doit, false)
+    end
+    markers = list_of_markers(harrow_markers, doit_markers)
+    [markers, doit_orgs]
+  end
+
+  def list_of_markers(harrow_markers, doit_markers)
+    return doit_markers if harrow_markers == '[]' 
+    return harrow_markers if doit_markers == '[]' 
+    harrow_markers[0...-1]+', ' + doit_markers[1..-1]
+  end
+
+  def build_map_markers(organisations, type = :harrow, include_extra_organisation_data = true)
```

Comment:
Metrics/LineLength: Line is too long. [94/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Metrics/MethodLength: Method has too many lines. [16/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)